build_tools/circle/build_wheels_arm.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+set -x
+
+sudo add-apt-repository --remove ppa:ubuntu-toolchain-r/test
+sudo apt-get update
+sudo apt-get install python3-virtualenv ccache
```

Comment:
Once each build is running in its own circle ci docker image, we can just use the Python version pre-installed in the image instead and we would not need to create a venv anymore.

Reply:
I think we can also skip the ccache installation since the compiling happens inside isolated docker containers that won't use any host system command.

Reply:
done

Reply:
I kept the venv, as in ubuntu the default `python` command is `python3` , and I found the venv suite of commands more general... but no strong opinion on that.

Reply:
I agree.